packages/jest/src/generators/init/schema.d.ts

```diff
@@ -1,3 +1,3 @@
-export interface JestInitSchema {
-  babelJest?: boolean;
-}
+import { JestProjectSchema } from '@nrwl/jest/src/generators/jest-project/schema';
```

Comment:
suggestion: this should be a local path since we are inside the jest project already.